lib/chef/resource/aws_dhcp_options.rb

```diff
@@ -12,6 +12,7 @@
 #
 class Chef::Resource::AwsDhcpOptions < Chef::Provisioning::AWSDriver::AWSResourceWithEntry
   aws_sdk_type AWS::EC2::DHCPOptions
+  include Chef::Provisioning::AWSDriver::AWSTaggable
```

Comment:
This is what each resource that wants the `aws_tags` attribute will look like and include